torch_geometric/loader/link_neighbor_loader.py

```diff
@@ -288,7 +280,6 @@ def __init__(
         # Save for PyTorch Lightning < 1.6:
         self.num_neighbors = num_neighbors
         self.edge_label_index = edge_label_index
-        self.edge_label = edge_label
```

Comment:
Why remove?

Reply:
`self.edge_label` isn't used anywhere.

Reply:
Yes, but PyTorch Lightning still expects that it is present.

Reply:
Ah good point. added it back.